cmake.deps/CMakeLists.txt

```diff
@@ -163,9 +163,19 @@ set(LUV_SHA256 16720d17b9e6d42a7008b902207ea2d61e3eeaef3cf358398dbbf3777867ec09)
 set(LUA_COMPAT53_URL https://github.com/keplerproject/lua-compat-5.3/archive/v0.9.tar.gz)
 set(LUA_COMPAT53_SHA256 ad05540d2d96a48725bb79a1def35cf6652a4e2ec26376e2617c8ce2baa6f416)
 
-# cat.exe curl.exe curl-ca-bundle.crt diff.exe tee.exe xxd.exe
-set(WINTOOLS_URL https://github.com/neovim/deps/raw/c1e7dd8de9e1b18d11dcfa0a192cd029262e5303/opt/win32tools.zip)
-set(WINTOOLS_SHA256 3c4c490a3d392ceeb1347cb77cc821a31900b688a2189276d3a1131a3f21daf1)
+# Windows only: cat.exe curl.exe curl-ca-bundle.crt diff.exe tee.exe xxd.exe
+set(CAT_URL https://github.com/dundargoc/deps/raw/master/opt/cat.exe)
+set(CAT_SHA256 93b8d307bb15af3968920bdea3beb869a49d166f9164853c58a4e6ffdcae61c6)
+set(CURL_CA_BUNDLE_URL https://github.com/dundargoc/deps/raw/master/opt/curl-ca-bundle.crt)
+set(CURL_CA_BUNDLE_SHA256 6ed95025fba2aef0ce7b647607225745624497f876d74ef6ec22b26e73e9de77)
+set(CURL_URL https://github.com/dundargoc/deps/raw/master/opt/curl.exe)
+set(CURL_SHA256 4c7151dea34c12b351030d4a14e50336a5a4e4639726565829997eea0d744654)
```

Comment:
I vaguely recall curl involving several files. I assume you checked that `curl` works on windows with only these 2 files?